styles/base/inputs/_input-container.scss

```diff
@@ -230,43 +235,18 @@
 }
 
 .sprk-b-InputContainer--huge
-  > :first-child.sprk-b-TextInput:disabled.sprk-b-TextInput--label-hidden {
+  > :first-child.sprk-b-TextInput:disabled.sprk-b-TextInput--label-hidden,
+.sprk-b-InputContainer--huge .sprk-b-TextInputIconContainer .sprk-b-TextInput:disabled.sprk-b-TextInput--label-hidden {
   color: $sprk-text-input-disabled-color;
 }
 
 .sprk-b-InputContainer--huge
-  > :first-child.sprk-b-TextInput:not(:placeholder-shown) {
+  > :first-child.sprk-b-TextInput:not(:placeholder-shown),
+.sprk-b-InputContainer--huge .sprk-b-TextInputIconContainer .sprk-b-TextInput:not(:placeholder-shown) {
   padding-top: $sprk-input-huge-active-padding-top;
 }
 
-// Huge styles for text inputs with icon
```

Comment:
These were all a first attempt when I did the work way back when. None of these selectors were accurate and don't actually do anything. So, DELETED.